package.json

```diff
@@ -64,13 +77,15 @@
   "devDependencies": {
     "babel-runtime": "5.8.29",
     "babelify": "6.4.0",
-    "browserify": "6.1.0",
+    "browserify": "10.0.0",
+    "browserify-derequire": "^0.9.4",
```

Comment:
Yeah let's peg at the current versions (we can make an effort to review and update packages every few months). So this one and the other new ones (budo, karma-browserify, etc.). Thanks!